### PR TITLE
Refactor manual components

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Finally, we apply the new markup and style to the client-side manual components. Lots of additions in this PR but this will be quickly followed by a lot of reduction in a forthcoming PR, once we are confident the new components are safe to go online.

/cc @uplne @jbreckmckye @Calanthe

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/creatives/template-preprocessor.js [hunk 1]
```diff
@@ -134,11 +151,60 @@ define([
         }
     }
 
+    function preprocessManualContainer(tpl) {
+        manualContainerButtonTpl || (manualContainerButtonTpl = template(manualContainerButtonStr));
+        manualCardTpls[tpl.params.creativeCard] || (manualCardTpls[tpl.params.creativeCard] = template(manualCardStrs[tpl.params.creativeCard]));
+        tpl.params.classNames = tpl.params.classNames.map(function (cn) { return 'adverts--' + cn; }).join(' ');
+        tpl.params.title || (tpl.params.title = '');
+        if (tpl.params.isSoulmates) {
```
Comment (uplne) [inline static/src/javascripts/projects/common/modules/commercial/creatives/template-preprocessor.js]: Can you put some spaces in these madness?

File: static/src/javascripts/projects/common/modules/commercial/creatives/template-preprocessor.js [hunk 2]
```diff
@@ -134,11 +151,60 @@ define([
...
+            manualContainerCtaSoulmatesTpl || (manualContainerCtaSoulmatesTpl = template(manualContainerCtaSoulmatesStr));
+            tpl.params.title = tpl.params.marque54icon + tpl.params.logosoulmates + '<span class="u-h">The Guardian Soulmates</span>';
+            tpl.params.blurb = 'Meet someone <em>worth</em> meeting';
+            tpl.params.ctas = manualContainerCtaSoulmatesTpl(tpl.params);
+        } else if (tpl.params.isMembership) {
```
Comment (uplne) [inline static/src/javascripts/projects/common/modules/commercial/creatives/template-preprocessor.js]: Have you thought about some switch or map on this? It may increase readability.

File: static/src/stylesheets/module/commercial/_adverts-soulmates.scss [hunk 3]
```diff
@@ -121,6 +130,27 @@
     }
 }
 
+.advert--soulmates {
+    .button {
```
Comment (uplne) [inline static/src/stylesheets/module/commercial/_adverts-soulmates.scss]: What about `.advert__btn--soulmates` etc?

Comment (regiskuckaertz) [inline static/src/stylesheets/module/commercial/_adverts-soulmates.scss]: That's sth I'd rather do once and for all on a separate PR if you don't mind.

File: static/src/javascripts/projects/common/modules/commercial/creatives/template-preprocessor.js (hunk 1 above)
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/commercial/creatives/template-preprocessor.js]: Yes!

File: static/src/javascripts/projects/common/modules/commercial/creatives/template-preprocessor.js (hunk 2 above)
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/commercial/creatives/template-preprocessor.js]: How? 😄

Comment (uplne) [inline static/src/javascripts/projects/common/modules/commercial/creatives/template-preprocessor.js]: Oh yeah true

**Comment by uplne** [comment]
👍
